### ISSUE TITLE
Add ability to easily re-run finished experiment

### ISSUE THREAD

**Issue by john-thomas-dotcms** [dotCMS/core#28153]
## User Story

As a marketing user, I want to be able to re-run an already completed experiment, to collect additional data.

Source of request: dotCMS internal Marketing team.

## Requirements

* Copy the experiment configuration and all variants into a new experiment with new variants.
    * Set the name of the new experiment to the same name as the prior experiment with an incremental number at the end, starting with 2.
    * Set the names of the new variants to exactly match the names of the variants in the prior experiment.
    * Set the Goal and all its parameters to exactly match the old experiment.
    * Do not copy the schedule from the old experiment to the new one.
* Set the prior for the new experiment to the results of the prior experiment.
    * Note: This issue is thus dependent on #26052 .

### Out of Scope

* The data from the old experiment and the new experiment will NOT be merged.
    * This means you will not be able to view the data from both experiments in the same graph or the same table.
    * But it also means you will be able to compare the data from the experiments.
* There will not be any way to compare the old and new experiment directly.
    * You'll only be able to compare the results as you would any other different experiments.
    * Adding a more direct comparison could be another feature to be implemented at a later date.

## Acceptance Criteria

After an experiment has ended, you can re-run the experiment by taking just the following steps:
1. Select "Re-run the Experiment" on the experiment in question.
2. Set the Schedule for the new experiment, or set it to start immediately.

## Dependencies
#26052
